flashinfer/comm/trtllm_alltoall.py

```diff
@@ -309,6 +402,25 @@ def get_moe_workspaces(mapping: Mapping):
         )
         return MnnvlMoe.moe_workspace_tensor
 
+    @staticmethod
+    def get_moe_prepare_workspace(
+        mapping: Mapping, config: Optional[MnnvlConfig] = None
```

Comment:
`MnnvlConfig` is not defined

Reply:
These are added in https://github.com/flashinfer-ai/flashinfer/pull/1245